src/interpreter/executors/instruction.rs

```diff
@@ -240,16 +242,6 @@ where
                 self.alu_set(ra, b ^ imm)?;
             }
 
-            OpcodeRepr::CIMV => {
```

Comment:
ideally we remove these in a separate PR to make it easier to track what changes were made in each release.

Reply:
We have this pr: https://github.com/FuelLabs/fuel-vm/pull/158

But its blocked by the fuel-tx one: https://github.com/FuelLabs/fuel-tx/pull/149